_rules/SC1-1-1-image-has-name.md

```diff
@@ -19,21 +19,21 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML element with the [semantic role][] of `image` that is [exposed to assistive technologies][].
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `image` that is [exposed to assistive technologies](#exposed-to-assistive-technologies).
 
-**NOTE**: Any image marked as decorative, by using either `role="presentation"`, `role="none"` or an empty `alt` attribute in combination with either no `title` attribute or an empty `title` attribute, does not have the role of `image` according to [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+**NOTE**: Any image marked as decorative, by using either `role="presentation"`, `role="none"` or an empty `alt` attribute, does not have the role of `image` according to [ARIA in HTML](https://www.w3.org/TR/html-aria/#img-alt).
 
 ### Expectation
 
-Each target element has an [accessible name][] that is [non-empty][].
+Each target element has an [accessible name](#accessible-name) that is [non-empty](#non-empty).
 
 ## Assumptions
 
 _There are currently no assumptions_
 
 ## Accessibility Support
 
-Certain assistive technologies can be set up to ignore the `title` attribute, which means that to some users the `title` attribute will not act as an accessible name.
+Certain assistive technologies can be set up to ignore the `title` attribute, which means that to some users the `title` attribute will not act as an [accessible name](#accessible-name).
```

Comment:
Either `title` should be considered as not supported as removed as a test case, or be included in the 'accessible name' algorithm.